lib/EditCard/EditCard.js

```diff
@@ -16,16 +15,37 @@ export default class EditCard extends React.Component {
     deleteBtnProps: {},
   }
 
-  renderDeleteButton = () => (
-    <Button
-      buttonStyle="link slim"
-      style={{ margin: 0, padding: 0 }}
-      onClick={this.props.onDelete}
-      {...this.props.deleteBtnProps}
-    >
-      <Icon icon="trash" />
-    </Button>
-  )
+  renderDeleteButton = () => {
+    const { deleteButtonTooltipText } = this.props;
+    return (
+      deleteButtonTooltipText ?
+        (
+          <Tooltip
+            text={deleteButtonTooltipText}
+          >
+            {({ ref, ariaIds }) => (
+              <IconButton
```

Comment:
Can you clean this section up a bit? In one case we're rendering an IconButton and in the other a `<Button><Icon /></Button>`. Go ahead and just use an IconButton all the time since it looks like that way we avoid the blue colour that Gill requested we remove.